Assets/UGSSamples/Plugins/PlayerProfile/SampleAuthenticator.cs

```diff
@@ -0,0 +1,45 @@
+
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Unity.Services.Authentication;
+using Unity.Services.Core;
+using UnityEngine;
+
+namespace Unity.Services.Samples
+{
+    /// <summary>
+    /// The Drag & Drop Samples shared implementation of the Unity Authentication Service
```

Comment:
Is it a good idea to call our samples "Drag & Drop" ?

Reply:
No, its just been stuck in my mind from talking about the drag & drop feature of **this** sample